### PR TITLE
fix: adds empty search state

### PR DESCRIPTION
**Pull request by mikaelbr**
Fixes empty state / default state for assistant and departure views.

Screenshot 2023-11-22 at 09 28 45

### CODE REVIEW

File: src/components/empty-search/index.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import { motion } from 'framer-motion';
+import { PropsWithChildren } from 'react';
+import { Image } from '@atb/components/icon';
+
```
Comment (adriansberg) [inline src/components/empty-search/index.tsx:4]: Any logic behind separation of imports?

File: src/components/empty-search/index.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  const hasEmptyChild = React.Children.toArray(children).some((child) => {
+    if (React.isValidElement(child)) {
+      return 'empty' in child.props;
+    }
+    return false;
+  });
+
+  return (
+    <>
+      {!isSearching && hasEmptyChild && (
```
Comment (adriansberg) [inline src/components/empty-search/index.tsx:31]: Would it be clean to bake this into a switch?

File: src/components/empty-search/index.tsx (hunk 1 above)
Comment (mikaelbr) [reply src/components/empty-search/index.tsx]: Copy paste issue. I like to separate css from the others.

Comment (adriansberg) [reply src/components/empty-search/index.tsx]: I'm on board with that.

File: src/components/empty-search/index.tsx (hunk 2 above)
Comment (mikaelbr) [reply src/components/empty-search/index.tsx]: I think when extending to error / geolocation we should bake this into type system

**Pull request merged**